android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -1999,6 +1999,9 @@ public static int getCurrentKeyboardIndex(Context context) {
 
   public static Keyboard getCurrentKeyboardInfo(Context context) {
     int index = getCurrentKeyboardIndex(context);
+    if(index < 0) {
+      return null;
+    }
```

Comment:
This is the primary fix. I audited all calls to `getCurrentKeyboardInfo`, and where a `null` result was not tested, fixed that as well.